### ISSUE TITLE
Show blog category on blog archive and blog detail pages

### ISSUE THREAD

**Issue by sboerrigter** [trendwerk/nextjs-starter#66]

**Comment by haroldangenent** [comment]
I believe this is about showing which category a posts belongs to. I've created a different issue for showing the categories on blog detail pages: #72.

**Comment by sboerrigter** [comment]
Yep, I meant the blog category name in the post meta (next to the post date for example).
